tutorials/emojimon/step-11.md

```diff
@@ -12,10 +12,10 @@ Since MUD handles all the networking for us, and because we're using a common ba
 
 We're currently only rendering our own player. To make the game multiplayer, we just need to query for other players and render them too!
 
-```tsx !#1-2,5,9,16-26,37-39,53-58 packages/client/src/GameBoard.tsx
-import { useEffect, useMemo, useState } from "react";
+```tsx #2,5,9,16-24,33,47-52 packages/client/src/GameBoard.tsx
```

Comment:
Do we need the `!` here? I forget if `!` hides or shows the line numbers

Reply:
oh yep, good catch